src/cli/commands/inbox.py

```
@@ -362,12 +362,133 @@ def _format_size(size: int) -> str:
+                console.print(f"  - {pdf.name} ({size_str})")
+            console.print(f"\nTotal: {len(pdf_files)} file(s)")
+            return
+
+        # Initialize Reducto client
+        try:
+            reducto_config = ReductoConfig.from_env()
+            reducto_config.agentic = agentic
+        except ValueError as e:
+            console.print(f"[red]Error: {e}[/red]")
+            console.print("Set your API key: export REDUCTO_API_KEY=your_key_here")
+            raise typer.Exit(1)
```

Ruff (0.14.10)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)